criu/net.c

```diff
@@ -3053,6 +3054,54 @@ static int iptables_restore(bool ipv6, char *buf, int size)
 	return ret;
 }
 
+static inline int nftables_lock_network_internal(void)
+{
+#if defined(CONFIG_HAS_NFTABLES_LIB_API_0) || defined(CONFIG_HAS_NFTABLES_LIB_API_1)
+	struct nft_ctx *nft;
+	int ret = 0;
+
+	nft = nft_ctx_new(NFT_CTX_DEFAULT);
+	if (!nft)
+		return -1;
+
+	if (NFT_RUN_CMD(nft, "add table inet CRIU")) {
+		ret = -1;
+		goto out;
+	}
+
+	if (NFT_RUN_CMD(nft, "add chain inet CRIU output { type filter hook output priority 0; policy drop; }")) {
+		ret = -1;
+		goto err;
+	}
+
+	if (NFT_RUN_CMD(nft, "add rule inet CRIU output meta mark " __stringify(SOCCR_MARK) " accept")) {
+		ret = -1;
+		goto err;
+	}
+
+	if (NFT_RUN_CMD(nft, "add chain inet CRIU input { type filter hook input priority 0; policy drop; }")) {
+		ret = -1;
+		goto err;
+	}
+
+	if (NFT_RUN_CMD(nft, "add rule inet CRIU input meta mark " __stringify(SOCCR_MARK) " accept")) {
+		ret = -1;
+		goto err;
+	}
+
+	goto out;
+
+err:
+	NFT_RUN_CMD(nft, "delete table inet CRIU");
+out:
+	nft_ctx_free(nft);
+	return ret;
+#else
+	pr_err("CRIU must be compiled with libnftables support\n");
```

Comment:
I am not sure I understand clearly, Do you mean the message should appear when CRIU is installed from a package manager?

Reply:
No, I think that we could make the following change:
```diff
-pr_err("CRIU must be compiled with libnftables support\n");
+pr_err("CRIU was built without libnftables support\n");
```
The reason is because if CRIU is installed from a package (e.g. `apt-get install criu`) this message may appear to a user.